ra-optimisation/search-tree-rao/src/main/java/com/farao_community/farao/search_tree_rao/linear_optimisation/algorithms/IteratingLinearOptimizer.java

```diff
@@ -57,7 +56,9 @@ public LinearOptimizationResult optimize(LinearProblem linearProblem,
             } else if (linearProblem.getStatus() != LinearProblemStatus.OPTIMAL) {
                 BUSINESS_LOGS.error("Linear optimization failed at iteration {}", iteration);
                 if (iteration == 1) {
-                    return new FailedLinearOptimizationResultImpl();
+                    bestResult.setStatus(LinearProblemStatus.INFEASIBLE);
+                    BUSINESS_LOGS.info("Linear problem is infeasible, initial situation is kept.");
```

Comment:
use 'linearProblem.getStatus()' in these 2 lines instead (can be different from infeasible)